src/drive/web/modules/views/OnlyOffice/View.jsx

```diff
@@ -6,12 +6,9 @@ import Spinner from 'cozy-ui/transpiled/react/Spinner'
 import Error from 'drive/web/modules/views/OnlyOffice/Error'
 import { OnlyOfficeContext } from 'drive/web/modules/views/OnlyOffice'
 
-const forceIframesHeight = value => {
-  const iframes = document.getElementsByTagName('iframe')
-
-  for (const iframe of iframes) {
-    iframe.style.height = value
-  }
+const forceIframeHeight = value => {
+  const iframe = document.getElementsByName('frameEditor')[0]
```

Comment:
All the stuff related to the DOM structure of OO should be centralized somewhere. The idea is to have a quick and clear vision on what we are relying to prevent BC when upgrading OO version.